src/store/index.js

```diff
@@ -0,0 +1,30 @@
+import Vue from 'vue';
+import Vuex from 'vuex';
+
+// import { createPersistedState, createSharedMutations } from 'vuex-electron';
+
+import modules from './modules';
+
+import migrations from './migrations';
+
+Vue.use(Vuex);
+
+const store = new Vuex.Store({
+  modules,
+  plugins: [
+    // location: AppData\Roaming\Electron\vuex.json
+    // createPersistedState(),
+    // createSharedMutations(),
```

Comment:
currently the changes to the store wont be saved to file

i need to check how to solve that

Reply:
Please see #30

Reply:
Ok, switched to  `vuex-persistedstate` but not using the cookies storage.

does it matter if the data stored at local storage or cookies?

Reply:
I hope you researched the two options well.

Because they are just options, I don't know how much they fit us.
For example, is `vuex-persistedstate` support the Electron needs at all?

The answer to this question will also answer your question, because if Electron works with cookies or local storage **and save them between sessions**, you can use them.
But I don't know which one is better. We need to save a big object, like a long list of transactions. Is there a problem saving large objects on cookies or local-storage?

Reply:
the [`vuex-persistedstate`](https://github.com/robinvdvleuten/vuex-persistedstate) package saves and read from the local storage by default, so it is easier to integrate it into our app.

the[`electron-store`](https://github.com/sindresorhus/electron-store) lets you store JSON Objects to file so the data will be saved to disk but then you need to update the vuex store on every change.

there is an advantage to `electron-store` like mentioned [`here`](https://github.com/sindresorhus/electron-store/issues/17#issuecomment-307561288)

there is a limitation of 10MB on local storage vs 4096 bytes on cookies.

Reply:
Hi, let's discuss it in #30 .
Can you copy your comment to #30, and I will answer you there?

Reply:
Hi, are you sure the file not saved with the current plugin?
Maybe it saved in another place? (Try [Everything](https://www.voidtools.com/))

I prefer to change the plugin in another PR, if we can (But don't waste time on it)

Reply:
I think it is a good idea to change the plugin in separate PR because we need to think about how to get the data that already stored into the new plugin.

the current plugin should work, but there is a problem on some getter

Reply:
OK, so the problem was that I've changed the name of the app by mistake.
there isn't a problem with `vuex-electron`